pkg/frontend/session.go

```diff
@@ -166,6 +168,18 @@ func (bgs *BackgroundSession) Close() {
 		bgs.cancel()
 	}
 }
+func (ses *Session) GenNewStmtId() uint32 {
+	if ses.lastStmtId == math.MaxUint32 {
+		// reused old id
+		ses.lastStmtId = 0
```

Comment:
I think MaxUint32 is a huge value for general usage. Do not need the check here.